src/Bridges/Constraint/bridges/geomean.jl

```diff
@@ -332,9 +332,14 @@ function _get_attribute(model, attr, bridge::GeoMeanBridge{T}) where {T}
     return output
 end
 
+function MOI.supports(::MOI.ModelLike, ::MOI.ConstraintPrimalStart, ::GeoMeanBridge)
+    @show @__LINE__
+    return true
```

Comment:
This is not true. It needs to check if the `model` supports it as well

Reply:
We had the reverse discussions in your PRs, the `AbstractBridgeOptimizer` takes care of that

Reply:
No, I don't think it does: https://github.com/jump-dev/MathOptInterface.jl/pull/2340

Is this the method you're thinking of? It just calls the bridge.
https://github.com/jump-dev/MathOptInterface.jl/blob/8b4b3ca384c22250e7e29bd90685e8d907a7e9f5/src/Bridges/bridge_optimizer.jl#L1474-L1527

It does not loop through the various constraints added.

Reply:
Ah but then I was wrong hence https://github.com/jump-dev/MathOptInterface.jl/pull/2339 etc... should be updated

Reply:
I already did :smile: https://github.com/jump-dev/MathOptInterface.jl/pull/2340

Reply:
I didn't follow that, looks good :)